cmp/chart/impl/copyToClipboard.js

```diff
@@ -0,0 +1,158 @@
+/*
+ * This file belongs to Hoist, an application development toolkit
+ * developed by Extremely Heavy Industries (www.xh.io | info@xh.io)
+ *
+ * Copyright © 2021 Extremely Heavy Industries Inc.
+ */
+import {XH} from '@xh/hoist/core';
+import {merge} from 'lodash';
+
+/**
+ * Copy the chart in it's current state to the clipboard.
+ * Works only on webkit based browsers.
+ */
+export function installCopyToClipboard(Highcharts) {
+    if (!Highcharts) return;
+    const  {Chart, extend} = Highcharts;
+
+    extend(Chart.prototype, {
+        copyToClipboard: async function(exportingOptions, chartOptions) {
```

Comment:
put implementation function also outside of the extend, for clarity.

Reply:
done.